lib/editor-settings.php

```diff
@@ -24,6 +24,8 @@ function gutenberg_extend_post_editor_settings( $settings ) {
 		$settings['defaultTemplatePartAreas'] = get_allowed_block_template_part_areas();
 	}
 
+	$settings['__experimentalLocalAutosaveInterval'] = 15;
```

Comment:
It might be good while we're here (sorry lol) to split this hook up into different `lib/compat` files so that we know which settings supposed to be copied over into Core come release time.

```php
// lib/compat/wordpress-5.9/block-editor-settings.php
add_filter( 'block_editor_settings_all', function( $settings ) {
	$settings['__unstableEnableFullSiteEditingBlocks'] = current_theme_supports( 'block-templates' );
	if ( wp_is_block_theme() ) {
 		$settings['defaultTemplatePartAreas'] = get_allowed_block_template_part_areas();
 	}
	return $settings;
} );

// lib/compat/wordpress-6.0/block-editor-settings.php
add_filter( 'block_editor_settings_all', function( $settings ) {
	$settings['__experimentalLocalAutosaveInterval'] = 15;
	return $settings;
} );
```

Reply:
Good point, I forgot to do that. I've made those changes in [9ce49a3](https://github.com/WordPress/gutenberg/pull/39180/commits/9ce49a33a7f5b1b4eef56e2220ddcd5cfca765e5) and [0c643cc](https://github.com/WordPress/gutenberg/pull/39180/commits/0c643cc81890a933b708c6db2385a3eca2977afb)